### ISSUE TITLE
Name of role/user is not quoted in DDL

### ISSUE THREAD

**Issue by mkgrgis** [dbeaver/dbeaver#11679]
#### System information: 
- DBeaver version
The last
- Additional extensions
No

#### Connection specification:
- Database name and version
PostgreSQL, Firebird, Oracle, maybe some other

#### Describe the problem you're observing:

After
```
CREATE USER "John Doe";
CREATE USER "Вася Пупкин";
```
See DDL for role/user. If there is space in name of role or user, there is no quoting => invalid DDL.

**Comment by LonwoLonwo** [comment]
Hello @mkgrgis 
As I can see, you totally right with PostgreSQL. I will fix it.
But I don't see user/role DDL in Oracle or Firebird. Tell me, please, where did you find them?

**Comment by mkgrgis** [comment]
Thank for note, @LonwoLonwo!
Firebird, `CREATE USER` see [this url](https://firebirdsql.org/refdocs/langrefupd25-security-sql-user-mgmt.html). No problems with 
```
PW="password"; DB_adr="/tmp/テスト.fdb"; U="Вася Пупкин"; Host="localhost";
/usr/bin/isql-fb -ch UTF8 "$Host:$DB_adr" -u "'$U'" -p "'$PW'";
Database: localhost:/tmp/テスト.fdb, User: Вася Пупкин
SQL> 
```
But in DBeaver there is no block of users for Firebird... Oh, I didn't see.

For Oracle the problem is around something like https://docs.oracle.com/cd/B19306_01/server.102/b14200/statements_6012.htm  But is there creating of `CREATE ROLE ...` for Oracle in DBeaver? I haven't use this for Oracle, but in DBeaver there is a list of roles in Oracle as list of schemas.

**Comment by LonwoLonwo** [comment]
yep, you it's true about User<>Schema. But we don't have user's DDL tab and as a schema, he is quoted. 

![2021-03-12 11_22_45-NEWTABLE - Persist Changes](https://user-images.githubusercontent.com/45152336/110913028-adedbf00-8325-11eb-961d-607e5f838c7f.png)

So. This bug is  for PostgreSQL.

**Comment by LonwoLonwo** [comment]
Fixed in 21.0.1

**Comment by uslss** [comment]
verified
